package.json

```diff
@@ -36,6 +36,7 @@
     "react-dom": "^16.8.4",
     "react-i18next": "^7.12.0",
     "react-input-mask": "2.0.4",
+    "react-table": "^6.9.2",
```

Comment:
I might would argue that since version 7 is out we should go for it.. I know its is alpha but lets at least have this on the table for discussion.

Reply:
What are the arguments for using the alpha version?

In general we should use the latest stable version of depdendencies. v7 is alpha and under heavy development so I would strongly recommend we don't use it. We are likely to run into many bugs as it's a complete re-write, and 6 is nice and stable. For v7 there are not even docs in the Readme

Reply:
Okay read through their issues. Apparently v7 will be out soonish.. v6 tho is not actively maintained anymore.. Hope we dont run into blockers with this :)